platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/jasperdb/files/hashmap/VirtualKeySetSerializer.java

```diff
@@ -32,7 +32,7 @@
  * purpose key serializer.
  */
 @ConstructableIgnored
-public class VirtualKeySetSerializer implements KeySerializer<VirtualKey<VirtualLongKey>> {
+public class VirtualKeySetSerializer implements KeySerializer<VirtualKey> {
```

Comment:
Could/should it be `implements KeySerializer<VirtualLongKey>`?

Reply:
Makes sense. Let me implement it